tests/cypress/integration/quickstarts_spec.js

```diff
@@ -6,7 +6,7 @@ describe('quickstarts page default selections spec', () => {
   const basePage = new BasePage();
 
   beforeEach(() => {
-    basePage.visit('/documentation/');
```

Comment:
Is this change necessary?

Reply:
Yes, since we moved away from the `yarn dev` step and instead started spinning up the server directly from the build, this test was failing because in `beforeEach` `/documentation` page was throwing 404 (expected but 404s are not handled well in cypress) but in `yarn dev` the page doesn't throw 404 but instead gives a 200 response. Anyway, the logic here was to just change the page to another page when we run a test in the file, which is achieved with visiting `/` route also.